### PR TITLE
Active users tests

### PR DESCRIPTION
**Pull request by noorary**
Added frontend, e2e tests and backend tests for active users and better backend testing for cumulative new users.

### CODE REVIEW

**Comment by noorary** [comment]
This can be merged as soon as someone has time to check and approve it, and if no issues/problems are found. Then 'show active users' userstory is done by DoD.

**Review by hanranti — APPROVED** [review]
This looks good!

**Pull request merged**
